.readthedocs.yaml

```diff
@@ -0,0 +1,32 @@
+# .readthedocs.yaml
+# Read the Docs configuration file
```

Comment:
Do we have a way to update the old docs to point to the new URL? Otherwise Google will point people at old docs and they won't know new ones exist.

Reply:
readthedocs allow custom domains. We could simply redirect the `eventlet.org` domain to the new readthedocs hosted instance. Every users would be automatically redirected.

Also, if necessary, we could publish an empty html index on the github page containing the following javascript instruction to redirect to the new docs hosted on readthedocs:

`window.location.href = ‘https://eventlet.readthedocs.io/’;`